Description: Disable global menus to workaround bug with 'ubuntu-local'
Author: Michael Terry <michael.terry@canonical.com>
Bug-Ubuntu: https://launchpad.net/bugs/743265

Index: freeciv-2.4.0/client/gui-gtk-2.0/gui_main.c
===================================================================
--- freeciv-2.4.0.orig/client/gui-gtk-2.0/gui_main.c	2013-09-14 23:00:05.617475589 +0200
+++ freeciv-2.4.0/client/gui-gtk-2.0/gui_main.c	2013-09-14 23:00:05.613475589 +0200
@@ -1529,6 +1529,10 @@
   /* GTK withdraw gtk options. Process GTK arguments */
   gtk_init(&argc, &argv);
 
+  /* Disable Ubuntu global menus as a temporary workaround for bug
+     https://launchpad.net/bugs/743265 */
+  g_setenv("UBUNTU_MENUPROXY", "", TRUE);
+
   /* Load resources */
   gtk_rc_parse_string(fallback_resources);
 
